authors/settings.py

```diff
@@ -138,6 +142,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
We're not dealing with any form of authentication at the moment. Perhaps we can leave out this part for now.

Reply:
Lemme scrap it out